examples/standalone/runfile/dynamics.py

```diff
@@ -165,7 +165,13 @@ def write_global_timings(experiment, filename, comm, root=0):
         input_data = driver_object.collect_input_data(serializer, savepoint_in)
         input_data["comm"] = communicator
         state = driver_object.state_from_inputs(input_data)
-        dycore = fv3core.DynamicalCore(communicator, spec.namelist)
+        dycore = fv3core.DynamicalCore(
+            communicator,
+            spec.namelist,
+            state["atmosphere_hybrid_a_coordinate"],
+            state["atmosphere_hybrid_b_coordinate"],
```

Comment:
aren't we still calling these ak and bk everywhere else? that is a little confusing, but good to specify what these things are

Reply:
In the state, meaning in the call interface to DynamicalCore/entrypoint to the FV3, they've always been called this. They get translated to ak and bk for use inside the dycore code. That interface was specifically to document what ak and bk mean using code, as you noted :)

I think we'll definitely want to clean up this interface when we do a clean-up of all the init interfaces, after making the initial set of objects. These feel like variables that should exist on some kind of "grid" object.